pkg/cmd/eventlistener/describe.go

```diff
@@ -81,6 +81,15 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .EventListener.Name }}
 {{- end }}
 {{ " " }}
 {{- end }}
+{{- if isBindingNameExist $v.Bindings }}
+  NAME	VALUE
+{{- range $b := $v.Bindings }}
+{{- if ne $b.Name "" }}
+  {{ decorate "bullet" $b.Name }}	{{ $b.Value }}
```

Comment:
Wasn't sure what to make of this, but the example [here](kubectl apply -f https://raw.githubusercontent.com/tektoncd/triggers/master/examples/eventlisteners/eventlistener-embedded-bindings.yaml) comes back with a `<nil>` for value:

```
Name:                      listener-embed-binding
Namespace:                 default
Service Account:           tekton-triggers-example-sa
URL:                       http://el-listener-embed-binding.default.svc.cluster.local:8080
EventListnerServiceName:   el-listener-embed-binding
 
EventListenerTriggers

 NAME
 ∙ foo-trig
 
 BINDINGS

  REF                  KIND   APIVERSION
  ∙ pipeline-binding          
 
  NAME                VALUE
  ∙ message-binding   <nil>
 
  SPEC
  ∙ Params
    message   Hello from the Triggers EventListener(listener-embed-binding)!
 
 TEMPLATE REF          APIVERSION
 ∙ pipeline-template 
```

I suppose we can leave it for now, but might be nice to clean up in future.

Reply:
definitely, we will need cleanup, as the deprecated part becomes unsupported.